app/assets/javascripts/components/signup/signup.js.coffee

```diff
@@ -80,7 +83,11 @@ modulejs.define 'components/signup/signup',
         ),
         (div {className: 'side-info'},
           if studentData
-            (StudentRegistrationCompleteSideInfo {})
+            # StudentRegistrationCompleteSideInfo contains a login form
+            # If the student is already logged in because of the SSO path, don't show
+            # this form
+            if anonymous
+              (StudentRegistrationCompleteSideInfo {})
```

Comment:
Probably it is correct, but double-checking: those nested ifs will result in nothing being rendered, not even SideInfo. That's intended, right?